RockLib.Configuration.AppSettingsConfigTests/ConfigTests.cs

```diff
@@ -28,7 +30,7 @@ public void EnvironmentVariablesOverrideFileConfig()
         {
             // Note that the "AppSettings:Environment" environment variable was
```

Comment:
Should probably look for "Environment" (without the "RockLib." prefix) in comments and replace with prefixed.

Reply:
Comments... get me every time :-/